src/frontend/package.json

```diff
@@ -15,7 +15,6 @@
     },
     "devDependencies": {
         "@iconify/vue": "^4.1.1",
-        "@nuxt/devtools": "^1.0.0",
```

Comment:
?? Dont like it anymore?

Reply:
devtools integrated with nuxt now, not external dep, I believe!

https://github.com/nuxt/nuxt/releases/tag/v3.8.0